Calculator.md

```diff
@@ -0,0 +1,99 @@
+# Brittany Jones
+# Ada Calculator
+# Last Edited 2/7/18
+# The program should ask the user for an operation (string or numeric symbol) and two numbers.
+
+#Primary Requirements
+#The program should use the input operation and two numbers to provide the result of applying the operation to the two numbers.
+#The program should have support for these four operations: addition, subtraction, multiplication, and division.
+#The program should accept both the name (add) and the symbol (+) for each possible operation.
+
+
+# Prompt user.
+puts "Let us do math!"
+
+# Get first number from user.
+puts "\nEnter the the first number:"
+a = gets.chomp
+while true
+    if a.to_f.to_s == a || a.to_i.to_s == a
```

Comment:
Works great, but I'd encourage you to make a loop that repeats while the input is wrong and not infinitely and then break out when the input is valid.  It just makes reading the code flow a little harder.  

Also `a` doesn't make a good variable name.  The name doesn't tell anyone what the variable would be used for.

Lastly you also indented too much here.  Indent by 1 tab or 2 spaces and be **consistent.**